e107_languages/English/admin/lan_admin.php

```diff
@@ -513,5 +513,8 @@
 define("LAN_LOCK", "Lock");
 define("LAN_SYSTEM_USER", "System user");
 define("LAN_ENGINE", "Engine");
-
+define("LAN_SOURCE", "Source");
+define("LAN_LAYOUT", "Layout");
+define("LAN_YEAR", "Year");
+	
```

Comment:
Thank you! Did you find other places in e107 which use "Source" and "Layout" and "Year" ?